### ISSUE TITLE
In examples, ignore >>> and ... when selecting code

### ISSUE THREAD

**Issue by chbndrhnns** [pawamoy/mkdocstrings#128]
**Is your feature request related to a problem? Please describe.**
I would like to be able to take examples and paste them in a python interpreter

```python
class A:
    """"
    Test

    Example:
    >>> import time
    >>>
    >>> print('something')
    """"
```

**Describe the solution you'd like**
In the rendered version of the class, I would like to omit the >>> and ... beginnings on the line as they break the Python interpreter.

This could be possible by using https://css-tricks.com/almanac/properties/u/user-select/

**Describe alternatives you've considered**
Are there other options?

**Additional context**

**Comment by pawamoy** [comment]
Makes sense. I'd recommend that you add the CSS with MkDocs' `extra_css` option though.

Two notes:
- ipython understands doctests, you can paste them directly inside the interpreter and it will work
- sections are supposed to be indented, did you mean

```python
class A:
    """
    Test

    Example:
        >>> import time
        >>>
        >>> print('something')
    """
```

?

**Comment by chbndrhnns** [comment]
Yes, it should have been indented.

I also see it works like that it ipython and I am often using the pycharm python console where it does not work:

<img width="1001" alt="Screenshot 2020-06-19 09 33 36" src="https://user-images.githubusercontent.com/7534547/85108467-0e752300-b210-11ea-9a9b-29dd4fefe1d5.png">

**Comment by pawamoy** [comment]
I think that PyCharm automatically uses ipython if you install it in the venv.

**Comment by oprypin** [comment]
You can do this, just write the example like this:

``````python
class A:
    """
    Test

    Example:
    ```pycon
    >>> import time
    >>>
    >>> print('something')
    ```
    """
``````

And add this CSS for Pygments, via `extra_css`.

```css
.highlight .gp, .highlight .go { /* Generic.Prompt, Generic.Output */
  user-select: none;
}
```

**Comment by pawamoy** [comment]
Nice @oprypin. I was also wondering last time: maybe we could write a Markdown extension to detect such code blocks anywhere, and not just in Examples sections.
